crates/symbolicator/src/metrics.rs

```diff
@@ -101,28 +101,26 @@ macro_rules! metric {
                 .send();
         })
     }};
-    (timer($id:expr), $block:block $(, $k:expr => $v:expr)* $(,)?) => {{
```

Comment:
The diff is a little messy here, but an unused type of timer here `metrics!(timer(..))` has been removed since it currently isn't being used anywhere in the codebase, as far as I can tell.